demo/src/plugins/dashboard/screens/dashboard/index.jsx

```diff
@@ -1,39 +1,62 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import ActiveEdits from '@triniti/cms/plugins/raven/components/active-edits-table';
 import NodeStatus from '@gdbots/schemas/gdbots/ncr/enums/NodeStatus';
 import { Col, Container, Row, Screen } from '@triniti/admin-ui-plugin/components';
 import TopArticles from '../../components/top-articles';
 
-const Dashboard = ({ location }) => (
+const Dashboard = ({ location, match: { params: { tab } } }) => (
   <Screen
-    header="Dashboard"
+    tabs={[
+      {
+        to: 'news',
+        text: 'news',
+      },
+      {
+        to: 'active-edits',
+        text: 'active edits',
+      },
+    ]}
     maxWidth="100%"
     title="Dashboard"
+    // NOTE: temporary fix until admin-ui solution for no breadcrumbs
+    breadcrumbs={{ length: null }}
     body={(
       <Container fluid className="dashboard">
-        <Row>
-          <Col lg="6">
-            <TopArticles
-              location={location}
-              title="Homepage Published"
-              search={{ count: 40, status: NodeStatus.PUBLISHED, slotting_key: 'home' }}
-            />
-          </Col>
-          <Col lg="6">
-            <TopArticles
-              location={location}
-              title="Homepage Draft"
-              search={{ count: 10, status: NodeStatus.DRAFT, q: '+is_homepage_news:true' }}
-            />
-          </Col>
-        </Row>
+        { tab === 'news'
+             && (
+             <Row>
+               <Col lg="6">
+                 <TopArticles
+                   location={location}
+                   title="Homepage Published"
+                   search={{ count: 40, status: NodeStatus.PUBLISHED, slotting_key: 'home' }}
+                 />
+               </Col>
+               <Col lg="6">
+                 <TopArticles
+                   location={location}
+                   title="Homepage Draft"
+                   search={{ count: 10, status: NodeStatus.DRAFT, q: '+is_homepage_news:true' }}
+                 />
+               </Col>
+             </Row>
+             )}
+        {tab === 'active-edits'
+             && (
+             <Row>
+               <Col lg="12">
+                 <ActiveEdits title="Active Edits" />
```

Comment:
@nanetteCodes default the title in the component to "Active Edits" so you don't have to set it here